apps/angular/src/app/layout/layout.component.html

```diff
@@ -0,0 +1,14 @@
+<mat-toolbar color="primary">
+  <a
+    mat-icon-button
+    routerLink="{{ routePaths.home }}"
+    class="toolbar__brand"
```

Comment:
If you have an *element* in the template (`toolbar__brand`), add *block* too, even if it would be an empty class
```suggestion
<mat-toolbar class="toolbar" color="primary">
  <a
    mat-icon-button
    routerLink="{{ routePaths.home }}"
    class="toolbar__brand"
```